manifest.json

```diff
@@ -415,20 +415,6 @@
         "rootDir": "ckeditor",
```

Comment:
Would changing the URL to `https://download.cksource.com/CKEditor/...`  help with people having SSL-related problems?